docs/extend/modular-resources/create/_index.md

```diff
@@ -493,15 +473,22 @@ For example:
 #!/bin/sh
 cd `dirname $0`
 
+# Create a virtual environment to run our code
+VENV_NAME="venv"
+PYTHON="$VENV_NAME/bin/python"
+
+python3 -m venv $VENV_NAME
+$PYTHON -m pip install -r requirements.txt -U # remove -U if viam-sdk should not be upgraded whenever possible
+
 # Be sure to use `exec` so that termination signals reach the python process,
 # or handle forwarding termination signals manually
-exec python3 -m <your-module-directory-name>.<main-program-filename-without-extension> $@
+exec $PYTHON src/main.py $@
 ```
 
 To make this shell script executable, run the following command in your terminal:
 
 ``` sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-sudo chmod +x <FILEPATH>/<FILENAME>
+sudo chmod +x <file-path-to-your-main-program>/<your-main-program-filename>
```

Comment:
```suggestion
sudo chmod +x <file-path-to-entrypoint>
```
maybe? the current text could imply `main.py`, when we want `run.sh`

Reply:
Oh yeah thank you!

Reply:
done